### PR TITLE
Make API credentials optional for Places usage

### PR DESCRIPTION
**Pull request by rayrutjes**
See https://github.com/algolia/places/pull/401 for more context.

### CODE REVIEW

File: tests/AlgoliaSearch/Tests/InitPlacesTest.php [hunk 1]
```diff
@@ -20,4 +20,20 @@ public function testExtraHeader()
...
+
+    /**
+     * @depends testShouldAllowToBeCalledWithoutCredentials
+     */
+    public function testCanSearchWithoutCredentials()
+    {
+        $index = Client::initPlaces();
+        $results = $index->search('Paris');
+        $this->assertArrayHasKey('nbHits', $results);
+        $this->assertGreaterThan(0, $results['nbHits']);
```
Comment (JanPetr) [inline tests/AlgoliaSearch/Tests/InitPlacesTest.php]: We gonna be screwed when Paris(es) got renamed :D :troll:

**Comment by raphi** [comment]
😚

File: tests/AlgoliaSearch/Tests/InitPlacesTest.php (hunk 1 above)
Comment (rayrutjes) [inline tests/AlgoliaSearch/Tests/InitPlacesTest.php]: Best joke of the day! Yet technically you are right :/

Comment (JanPetr) [inline tests/AlgoliaSearch/Tests/InitPlacesTest.php]: You can do empty query, that should always get some results, no?

Comment (rayrutjes) [inline tests/AlgoliaSearch/Tests/InitPlacesTest.php]: 👍  done

**Pull request merged**